### DISCUSSION TITLE
web server not working

### DISCUSSION THREAD

**Discussion by alfonder** [umputun/feed-master#122] [Q&A] [answered]
Just reinstalled FM from DockerHub image onto a new server. Waited for a couple hours to finish download-converting job. I see, that .mp3 and .xml files are generated. But I fail to get any data from web-server. All urls: /rss /feed /yt/<id> are returning error 404. And the following error is in log:
`{logger/logger.go:134} GET - /yt/UCWAIvx2yYLK_xTYD4F2mUNw - xxxxxxxx.dynu.net - unknown ip - 404 (19) - 27.297µs`
What does it mean? Previously my config worked on another server.

**Comment by umputun** [comment, marked as answer]
web server is working fine, probably some issue wit the configuration. In order to help you with this I need to see your docker-compose.xml and fm.xml as well as the full log generated from the moment you started the container.

_I'll move it to discussions as it doesn't seem to be the actual bug but rather something on your side_

**Comment by alfonder** [comment]
Thank you. I found a bug in my configuration. Now everything works as expected.
